spec/jobs/operation_job_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe OperationJob do
+  let(:operation) { create(:operation, job_class: subject) }
+
+  describe "#perform" do
+    subject(:operation_job) do
+      Class.new(described_class) do
+        def perform(operation, *args)
+        end
+      end
+    end
+
+    it "marks operation as started" do
+      subject.perform_now(operation)
+      expect(operation.reload.job_started_at).to_not(be_nil)
+    end
+
+    it "marks operation as completed" do
+      subject.perform_now(operation)
+      expect(operation.reload.job_completed_at).to_not(be_nil)
+    end
+
+    context "when error is raised" do
+      subject(:operation_job_with_error) do
+        Class.new(described_class) do
+          def perform(operation, *args)
+            raise StandardError
+          end
+        end
+      end
+
+      before { allow(ExceptionNotifier).to receive(:notify_exception).with(StandardError) }
+
+      it "marks operation as started" do
+        subject.perform_now(operation)
+        expect(operation.reload.job_started_at).to_not(be_nil)
```

Comment:
Can you please use `not_to` without parens? For some reason `to_not` is triggering a linter error for me.

I think because `to` and `not_to` are marked as special methods that don't need parens.